src/main/resources/application.conf

```diff
@@ -1,5 +1,5 @@
 dxWDL {
-    version = "v1.36"
+    version = "v1.37"
```

Comment:
I am going to start following semantic versioning. One of the issues is smart reuse. In other words, can we use a workflow compiled with an old dxWDL? Anyhow, please call this v1.36.1